src/libraries/Directory.Build.props

```diff
@@ -128,8 +128,9 @@
     <TestArchiveTestsDir>$(TestArchiveTestsRoot)$(OSPlatformConfig)/</TestArchiveTestsDir>
     <TestArchiveRuntimeRoot>$(TestArchiveRoot)runtime/</TestArchiveRuntimeRoot>
 
-    <ASPNETCoreAppPackageRefPath>$(ArtifactsBinDir)pkg\aspnetcoreapp\ref</ASPNETCoreAppPackageRefPath>
-    <ASPNETCoreAppPackageRuntimePath>$(ArtifactsBinDir)pkg\aspnetcoreapp\lib</ASPNETCoreAppPackageRuntimePath>
+    <ASPNETCoreAppPackageRootPath>$(ArtifactsBinDir)pkg\aspnetcoreapp\</ASPNETCoreAppPackageRootPath>
+    <ASPNETCoreAppPackageRefPath>$(ASPNETCoreAppPackageRootPath)ref</ASPNETCoreAppPackageRefPath>
+    <ASPNETCoreAppPackageRuntimePath>$(ASPNETCoreAppPackageRootPath)lib</ASPNETCoreAppPackageRuntimePath>
```

Comment:
nit: we usually make sure that paths that point to directories end with a slash by using the NormalizeDirectory functions.